cibuildwheel/options.py

```diff
@@ -310,17 +311,22 @@ def get(
             if table is None:
                 raise ConfigOptionError(f"{name} does not accept a table")
             return table["sep"].join(table["item"].format(k=k, v=v) for k, v in result.items())
-        elif isinstance(result, list):
+
+        if isinstance(result, list):
             if sep is None:
                 raise ConfigOptionError(f"{name} does not accept a list")
             return sep.join(result)
-        elif isinstance(result, int):
+
+        if isinstance(result, int):
             return str(result)
-        else:
-            return result
+
+        return result
 
 
 class Options:
+    # pre-decleration to avoid property decorator mypy clash.
```

Comment:
```suggestion
    # pre-declaration to avoid property decorator mypy clash.
```

Reply:
Not needed at all now that mypy knows what a `cached_property` is. :)